How to Find and Use Technical Resources if You’re New to Tech:proposal.md

```diff
@@ -0,0 +1,93 @@
+# Draft 3.0 - Finding Useful Coding Resources When You've Hit a Wall
+
+Knowing how to code is invaluable knowledge, no matter what level of mastery you're at. If you want to expand and strengthen your programming skills, awesome! But whether you've just started learning how to program or you've been doing this for years, you've likely run into some obstacles during your coding journey. 
+
+That's why we've compiled a list of tips on finding useful coding resources and some of our personal favorites. If you feel like you've hit a wall with your code, we hope you'll finding something that helps you out below! 
+
+## The Power of Googling
+
+Search engines are your best friends, and this is the number one way to find resources online because the internet is always there for you, no matter what time of day. 
+
+If you can't find what you're looking for right away, it can feel pretty frustrating. But it's important to not give up! 
+
+According to one of Bit Project's developers, Julia Ma, "One mistake I see a lot of  beginners make is that they don’t Google. When it comes to a problem that is coding-related — syntax, error, code compiling — always Google before you ask somebody because that’s how you learn the best."
+
+![](https://i.imgur.com/LyV2EDk.png)
+
+Sometimes you have to get specific with your search to get faster, better results. For example, you can get creative with what types of resources you look for: walkthroughs, documentation, [blog posts about coding](https://medium.com/topic/programming), etc. Or you can copy and paste the code you're having issues with directly into Google.
```

Comment:
Maybe say "you have to **be** specific" 
Also, maybe use a comma after 'Or' in the last sentence? I'm being picky so just do what you think sounds right.